### PR TITLE
filter sequence

### PR DESCRIPTION
**Pull request by cartersz**
### What problem does this PR solve?

Issue Number: ref #609 

### What is changed and how it works?
when get tables from schema, it should filter sequence

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

File: pkg/dbutil/common.go [hunk 1]
```diff
@@ -375,7 +375,7 @@ func GetTables(ctx context.Context, db QueryExecutor, schemaName string) (tables
 		| NTEST          | BASE TABLE |
 		+----------------+------------+
 	*/
-	query := fmt.Sprintf("SHOW FULL TABLES IN `%s` WHERE Table_Type != 'VIEW';", escapeName(schemaName))
+	query := fmt.Sprintf("SHOW FULL TABLES IN `%s` WHERE Table_Type != 'VIEW' and Table_Type != 'SEQUENCE';", escapeName(schemaName))
```
Comment (lichunzhu) [inline pkg/dbutil/common.go]: What about
```sql
	query := fmt.Sprintf("SHOW FULL TABLES IN `%s` WHERE Table_Type = 'BASE TABLE';", escapeName(schemaName))
```

Comment (lichunzhu) [inline pkg/dbutil/common.go]: What about
```sql
	query := fmt.Sprintf("SHOW FULL TABLES IN `%s` WHERE Table_Type = 'BASE TABLE';", escapeName(schemaName))
```

Comment (cartersz) [reply pkg/dbutil/common.go]: if "BASE TABLE" can include all user defined tables, i think it ok.

Comment (cartersz) [reply pkg/dbutil/common.go]: @lichunzhu done

Comment (cartersz) [reply pkg/dbutil/common.go]: @lichunzhu PTAL

**Review by lichunzhu — APPROVED** [review]
LGTM

File: pkg/dbutil/common.go (hunk 1 above)
Comment (cartersz) [reply pkg/dbutil/common.go]: lance6716 PTAL

**Review by Ehco1996 — APPROVED** [review]
LGTM

**Pull request merged**